### DISCUSSION TITLE
Access to /var/lib/docker?

### DISCUSSION THREAD

**Discussion by UnNamed-M3** [filebrowser/filebrowser#1559] [Q&A] [answered]
Hi,

I'm sure I'm doing something wrong but I'm not quite what... I have gone through the different documentation files but no luck so far. 

The goal is that I would like to use FileBrowser to view and edit all the files / folders even when they're in another container. 

I installed FileBrowser through Portainer's App Template and mapped the volumes as it should. This works just find but I can't seem to access files/folders in for example /var/lib/docker/. 

I though adding sudo su - under Settings -> User Management -> Commands would do the trick but it doesn't. 
I also tried editing the .filebrowser.json file with "Shell: sudo su -" but still no luck.

Could someone please provide the necessary step to achieve this?

**Comment by ramiresviana** [comment, marked as answer]
Running `su` command on the shell does not elevates the privileges of the current FIleBrowser instance, there's no reason to use that since docker runs the container as root by default.  Using a docker mount allows you to share files with the container, you will want to change the user scope or root option of FileBrowser to make those files accessible.

**Comment by UnNamed-M3** [comment]
Thanks for the quick feedback. Indeed by simply removing the "/" from the scope I was able to access all the files!
